ui/components/app/account-list-item/account-list-item.stories.js

```diff
@@ -4,7 +4,7 @@ import AccountListItem from './account-list-item';
 
 export default {
   title: 'Components/App/AccountListItem',
-  id: __filename,
+  id: 'ui-components-app-account-list-item-account-list-item',
```

Comment:
Could we just get rid of the `id` field altogether? From my understanding it does 2 things 
1. Creates the file path URL schema
2. Generates this section of links for each PR build 👇 

<img width="942" alt="Screenshot 2022-12-21 at 4 01 47 PM" src="https://user-images.githubusercontent.com/8112138/209026459-c82fe3c5-e267-47fc-b3f0-fefbbaaceba4.png">

🤔  I'm not sure it adds a huge amount of value because it seems to include stories that have no relevance to the PR https://github.com/MetaMask/metamask-extension/pull/16931/files.

Plus there is the other link to the storybook build so we could just use that for visual testing?

<img width="944" alt="Screenshot 2022-12-21 at 4 02 19 PM" src="https://user-images.githubusercontent.com/8112138/209026585-b6786e82-2364-444e-b1ae-83d3e248b070.png">

**Pros**
- Engineers don't have to fill out another field when creating a story

**Cons**
- Lose the ability to link to effected stories from the PR
- We would have to update all stories that link to other stories and think about externally links stories from Figma

So would have to update the links in Figma and in our MDX docs

![Screenshot 2022-12-21 at 3 55 46 PM](https://user-images.githubusercontent.com/8112138/209026913-5c36936c-2dc7-4359-92bd-a8ef2a42339f.png)

![Screenshot 2022-12-21 at 3 56 42 PM](https://user-images.githubusercontent.com/8112138/209026916-1f2ce402-3de3-48cf-885e-4787c569162a.png)

Thoughts on this @Gudahtt @kumavis?

Reply:
Yup, we can get rid of the id field altogether. @georgewrmarshall this is the current plan

1. We will delete the `id` field from all the stories.js files and won't be relying on `filename/id` anymore. 
2. Currently URLs are generated based on the existing ids but when the` ids `will be gone, new URLs will be generated based on the title of each story.
3. In the CI, we can get the URL from stories.json. 
4. The only con you mentioned will be to update the New URLs in the docs (README.mdx) files.